server/channels/app/slack.go

```
@@ -38,8 +39,13 @@ func (a *App) SlackImport(c request.CTX, fileData multipart.File, fileSize int64
 		},
 		GenerateThumbnailImage: a.generateThumbnailImage,
 		GeneratePreviewImage:   a.generatePreviewImage,
-		InvalidateAllCaches:    func() { a.ch.srv.InvalidateAllCaches() },
-		MaxPostSize:            func() int { return a.ch.srv.platform.MaxPostSize() },
+		InvalidateAllCaches: func() *model.AppError {
+			if err := a.ch.srv.InvalidateAllCaches(); err != nil {
+				return model.NewAppError("SlackImport", "app.slack.invalidate_cache.error", nil, err.Error(), http.StatusInternalServerError)
+			}
+			return nil
```

Can we just return the error from `InvalidateAllCaches`?